components/courses/spec/policies/courses/season/interviews_policy_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+describe Courses::Season::InterviewsPolicy do
+  let(:season)  { create(:season) }
+  let(:student) { create(:student) }
+  let(:policy)  { Courses::Season::InterviewsPolicy.new(season, student) }
+
+  describe '#allowed?' do
+    context "season has status registration, student has done a test task & student doesn't have an interview yet" do
+      it 'allows to pass policy' do
+        policy.stub(:season_has_status_registration?).and_return(true)
```

Comment:
stub it's rspec2 method, imho it's deprecated in rspec, try:
`allow(policy).to receive(: season_has_status_registration?).and_return(true)`